packages/app/components/Queue/StudentQueueList.tsx

```diff
@@ -130,6 +130,7 @@ export default function StudentQueueList({
         const newQuestions = [...questions, createdQuestion];
         await mutateQuestions(newQuestions);
         setPopupEditQuestion(true);
+        return true;
```

Comment:
this is the only actual change i'm not kidding

Reply:
dangothy